### PR TITLE
Update poker_get_players function in poker.py

### PR DESCRIPTION
**Pull request by phusitsom**

### CODE REVIEW

File: poker.py [hunk 1]
```diff
@@ -17,56 +17,52 @@
...
-
-            while True:
-                chk_msg = await client.wait_for("message", check=check)
-                if (chk_msg.content.lower() == 'y'):
-                    #print('len(players) =', len(players))
-                    return players
-                if (chk_msg.content.lower() == 'n'):
-                    await ctx.send('เอาให้เต็มที่ อย่าลืมพิมพ์ done นะ')
-                    break
-                else:
```
Comment (namirinz) [inline poker.py:68]: ?

Comment (phusitsom) [reply poker.py:68]: >:(

**Comment by namirinz** [comment]
Approved! Well done

**Review by namirinz — APPROVED** [review]
Approved! Well done : )

**Review by jiratQ — APPROVED** [review]
Well done Bro

**Pull request merged**